Console/UI/Pages/Options.xaml.cs

```diff
@@ -56,6 +56,7 @@ private void btnTestNotif_Click(object sender, RoutedEventArgs e)
 
         private void btnRestartAdmin_Click(object sender, RoutedEventArgs e)
         {
+            // TODO: @wokhan to be removed for power users?

```

Comment:
I still wonder if a non admin mode (as I implemented it) isn't still useful, in fact. Some asked for it...

Reply:
Maybe, but I don't know if it still works as you intended it to do, or has worked (before I touched it).
I'm honestly not sure how it was supposed to be used without admin rights (use case?) and then there is the question of how to test it properly (test case?).

Reply:
Many things were working (well, not so many, but at least the map, the adapters information... in fact every screen where I didn't add a "this screen requires the admin rights" message).
Regarding the use case, I think it was more user friendly to actually enable admin stuff when we need to. But I might be wrong;;;
Regarding the tests, the only way is to run it as non admin and see if admin stuff are indeed blocked ;)